docs/templates/github-actions/scan-and-fix/frogbot-scan-and-fix-go.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v2
```

Comment:
```suggestion
        uses: actions/setup-go@v3
```
same for others

Reply:
uses: actions/checkout@v2 >> uses: actions/checkout@v3

Reply:
In another PR that i will open i will update all the workflows including updating to v3 👍 
so i'm ignoring it for now here, and will address this in the follow-up PR.

Reply:
@sverdlov93  https://github.com/jfrog/frogbot/pull/340